modules/packages/ZMQ.chpl

```diff
@@ -165,13 +165,52 @@ versions may expose more features native to ZeroMQ.
 Serialization
 +++++++++++++
 
-In Chapel, sending or receiving messages on a :record:`Socket` uses
-`multipart messages <http://zguide.zeromq.org/page:all#Multipart-Messages>`_
-and the :chpl:mod:`Reflection` module to serialize primitive and user-defined
-data types whenever possible.  Currently, the ZMQ module serializes primitive
-numeric types, strings, and records composed of these types.  Strings are
-encoded as a length (as `int`) followed by the character array
-(in bytes).
+In Chapel, sending or receiving messages is supported for a variety of types.
+Primitive numeric types and strings are supported as the foundation.
+In addition, user-defined :type:`record` types may be serialized automatically
+as `multipart messages <http://zguide.zeromq.org/page:all#Multipart-Messages>`_
+by internal use of the :chpl:mod:`Reflection` module.
+Currently, the ZMQ module can serialize records of primitive numeric types,
+strings, and other serializable records.
+
+.. note::
```

Comment:
I think this is a fair nitpick. Does Chapel have module-level deprecation warnings? This also isn't strictly a deprecation, but a protocol change. With this change Chapel 1.14/15 programs using `ZMQ` can't talk to Chapel 1.16 programs, though they can talk intra-version just fine. I'm not sure who really uses old Chapel binaries with new Chapel binaries _and_ the `ZMQ` module. That set of potential users is almost certainly the empty set.

I think this is a case where the module versioning really has nothing to do with the Chapel release, but there isn't another release mechanism until `mason` comes online. I was trying to be explicit and clear about the changes in `ZMQ` from 1.15 to 1.16, since there isn't a change log or release notes for this (or any?) specific module.

If you still want it gone, I can take it out and tweak the wording as needed.